iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/OverloadImpl.java

```diff
@@ -42,6 +45,44 @@ public OverloadImpl(LoadingLimits.TemporaryLimit temporaryLimit, String previous
     }
 
     private static final LoadingLimits.TemporaryLimit UNACCEPTABLE_LIMIT = new LoadingLimits.TemporaryLimit() {
```

Comment:
This limit is a fictive one used only to report a constraint violation in a very specific case.
I think it would be better to not support properties on it. Therefore, you can throw an `UnsupportedOperationException` for the `setProperty` and `removeProperty` methods and return `null`/`false`/`Set.empty()` for the other methods.